homeassistant/components/waze_travel_time/sensor.py

```diff
@@ -248,6 +245,20 @@ def update(self):
 
         self._waze_data.update()
 
+    @property
+    def device_info(self) -> Optional[Dict[str, Any]]:
+        """Return device specific attributes."""
+        return {
+            "name": "Waze",
+            "identifiers": {(DOMAIN)},
```

Comment:
Is that the convention? I was going with the assumption that Waze should be treated as a single service because there are no API keys, etc. that would make these config entries unique from a Waze perspective

Reply:
Not sure. Most services seem to map against config entry unique_id or similar.

Reply:
If the choice was left to me, I would leave it as is. There's no value in having multiple device entries for Waze IMO. I have another PR to make this change for the Google Maps Time Travel one and in that one we create a device per API key because to me that's what makes it unique.

If you'd prefer that we go with unique_id, lmk, it's a simple enough change

Reply:
Let's keep it like this for now. I don't have a good reason for any choice besides other integrations.